src/app/beer_garden/api/http/handlers/v1/logging.py

```diff
@@ -71,7 +91,9 @@ async def patch(self):
         response = None
         for op in patch:
             if op.operation == "reload":
-                response = await self.client(Operation(operation_type="LOG_RELOAD"))
+                response = await self.client(
+                    Operation(operation_type="PLUGIN_LOG_RELOAD")
+                )
```

Comment:
Where do we invoke this API command? Because I can't find reference to it. They way I am reading this command is that if we change the plugin logging, if we call this endpoint then all local plugins get this update. But it does not work the same for remote plugins. They won't get it until restart.

Reply:
@TheBurchLog there's definitely room for improvement here. This reload operation probably shouldn't even exist - all it does is redo what the application does on startup. It doesn't send any updates to plugins, it only updates the configuration that will be returned if any new plugins ask.

Basically, if you want to change the configuration you'll need to update the file and then hit this endpoint (and we currently don't expose any way to hit this endpoint from the UI). After doing that any new plugins that come up will get the updated config.

We should probably just watch the plugin logging config file and execute this operation if it ever changes. If we do that then this API doesn't need to exist.